src/rise-lisk/commands/txs/createSignatureTx.c

```diff
@@ -37,19 +51,79 @@ static void stepProcessor_2nd_sign(uint8_t step) {
       deriveAddressStringRepresentation(address, lineBuffer);
       break;
     case 3:
-      for (i = 0; i < 3; i++) {
-        toHex(pubkey[i], lineBuffer + i * 2);
-      }
-      lineBuffer[6] = '.';
-      lineBuffer[7] = '.';
-      for (i = 0; i < 3; i++) {
-        toHex(pubkey[i + 32 - 3], lineBuffer + 8 + i * 2);
-      }
-      lineBuffer[14] = '\0';
+      truncatedHexPubKey(lineBuffer);
```

Comment:
you can use:
```
snprintf(lineBuffer, 50, "%.*H..%.*H",
              3, pubkey,
              3, pubkey + 32 - 3);
```

Reply:
Probably; I just moved the existing code into a new function so that it wouldn't be duplicated for the 2 UI targets we have. Since it's not directly related to NanoX support, I would exclude it from the scope of this PR.

But feel free to improve on the old code (that now lives in `truncatedHexPubKey` function) after this PR has been merged. Or create an issue for me and I'll do it in 3 weeks time.